libs/dal/src/repositories/job/job.repository.ts

```diff
@@ -207,6 +208,27 @@ export class JobRepository extends BaseRepository<JobDBModel, JobEntity, Enforce
       }
     }
 
+    if (filtered) {
+      await this._model.updateOne(
+        {
+          _environmentId: job._environmentId,
+          _templateId: job._templateId,
+          _subscriberId: job._subscriberId,
+          _id: job._id,
+        },
+        {
+          $set: {
+            status: JobStatusEnum.DELAYED,
```

Comment:
@ainouzgali if this one was filtered out, why not marking it as skipped instead?

Reply:
also then `digestResult: DigestCreationResultEnum.SKIPPED` should be updated ;)

Reply:
We can. But then we won't have the execution details. We will be having a different flow for digest than for the other steps.